db/migrate/20210620144826_add_purpose_to_notifications.rb

```diff
@@ -0,0 +1,13 @@
+class AddPurposeToNotifications < ActiveRecord::Migration[5.2]
```

Comment:
It may have been smarter to do the migration in its own PR rather than wrapping it up with a bunch of other code changes.

Reply:
It's simple enough, I think we can include it here. It also avoids the window of time where the column exists, but the code's not setting it.

Reply:
This is definitely the more correct thing thing to do. I didn't do it because I believe the only notifications that exist on prod are for medication reminders. I believe @prabhanshuguptagit confirmed that last week. I've implemented your suggestion here, but  before we run this migration on prod we should confirm that all notifications without subjects are medication reminders and that either 1) there are no notifications with subjects or 2) all notifications with subjects are appointment reminders. You can use the Notification#message attribute to distinguish them. I've already confirmed that there are no notifications in the db in Bangladesh.

Reply:
I think this reply was for the previous comment - https://github.com/simpledotorg/simple-server/pull/2613#discussion_r656844355

The less we have to keep in mind while shipping this, the better. Looks like we've updated the migration to operate on covid reminders and missed visit reminders separately. Nice 💯

Reply:
🤦 
Funny thing is I noticed your comment persisted after I made the change and thought you had left it at the top of the relevant code block so it wouldn't disappear when I changed the line in question.